packages/sit-onyx/src/components/OnyxNavItem/future/OnyxNavItem.vue

```diff
@@ -0,0 +1,75 @@
+<script setup lang="ts">
```

Comment:
question: is it intentional that the navItems do not yet emit any click event?

Reply:
added a click event